packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -497,34 +497,27 @@ const methods = {
 
     const hasMissingPatchesByHost = {}
     const hosts = filter(this.objects.all, { $type: 'host' })
-    await Task.run(
-      { properties: { name: `Listing missing patches`, total: hosts.length, progress: 0, done: 0 } },
-      async () => {
-        let done = 0
-        await asyncEach(hosts, async host => {
-          const hostUuid = host.uuid
-          await Task.run(
-            {
-              properties: {
-                name: `Listing missing patches for host ${hostUuid}`,
-                hostId: hostUuid,
-                hostName: host.name_label,
-              },
+    await Task.run({ properties: { name: `Listing missing patches`, total: hosts.length } }, async () => {
```

Comment:
IMHO, you can have both progress at the same time(`parent task progress` and `subtask progress`) instead of only keeping the `parent task progress`.
Same for other subtasks

Reply:
Ok, I removed the subtask progress because I thought it would not be useful as the only place it appears is in the task log.
Putting it back won't take much effort